### PR TITLE
Hometask 1

### PR DESCRIPTION
**Pull request by Shapovaleria**

### CODE REVIEW

File: middle_value.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function findMiddleValue (array) {
+  if (array instanceof (Array) == false || array.length == 0) {
```
Comment (StanislawVictorovich) [inline middle_value.js]: please separate statements and iterators with break-lines

File: sorted.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+function checkIfSorted(array) {
+  if (array instanceof (Array) == false || array.length == 0) {
+    return `function works with not empty arrays only`
+  }
+  for (let i = 0; i < array.length; i++) {
+    if (typeof (array[i]) != 'number') { return 'function works with numbers'};
```
Comment (StanislawVictorovich) [inline sorted.js]: please move return statement to the new line

File: middle_value.js (hunk 1 above)
Comment (Shapovaleria) [reply middle_value.js]: I'm not sure that I understood it correctly, but I tried. Done

File: sorted.js (hunk 2 above)
Comment (Shapovaleria) [reply sorted.js]: done

**Review by StanislawVictorovich — APPROVED** [review]

**Pull request merged**